package.json

```diff
@@ -48,6 +48,10 @@
     "rimraf": "^2.5.3"
   },
   "dependencies": {
+    "lodash.identity": "^3.0.0",
+    "lodash.isfunction": "^3.0.8",
+    "lodash.isnil": "^4.0.0",
+    "lodash.issymbol": "^4.0.1",
```

Comment:
This would be a new change, but I think it's a good one. Thoughts? 

Willing to roll our own to minimize dependencies, but why re-invent the wheel?